DemoApp/DemoApp/ViewModels/SwitchViewModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using Trimble.Modus.Components;
+
+namespace DemoApp.ViewModels
+{
+    public partial class SwitchViewModel : ObservableObject
```

Comment:
Can you follow naming convention here? `SwitchSamplePageViewModel`

Reply:
Yes, Renamed